### DISCUSSION TITLE
有关copilot

### DISCUSSION THREAD

**Discussion by mhxy13867806343** [community/community#56461] [Copilot Conversations]
### Select Topic Area

Question

### Body

只要在编辑器里面输入内容或者重新打开编辑在器的时候的问题 ，都会要我重新登录或者 在加载状态中，很烦 
<img width="844" alt="image" src="https://github.com/community/community/assets/4418166/77590bfb-8256-4881-8b17-f2c69b7c7304">

**Comment by Akash1134** [comment]
Hi @mhxy13867806343 , 

Thanks for participating! We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
